chart/values.yaml

```diff
@@ -2607,6 +2607,7 @@ config:
   celery:
     flower_url_prefix: '{{ ternary "" .Values.ingress.flower.path (eq .Values.ingress.flower.path "/") }}'
     worker_concurrency: 16
+    extra_celery_config: '{}'
```

Comment:
Similarly, you don't need to add this here - the default will take care of it.

Reply:
Removed this from values.yaml file. Added the default values in provider default yaml.